core-chart/templates/_helpers.tpl

```diff
@@ -1,8 +1,12 @@
 {{/*
 Expand ITS PCH custeradm container.
 */}}
+
+{{/*
+Container for initializing the cluster as an OCM hub using clusteradm init.
+*/}}
 {{- define "pch.its.custeradm" -}}

```

Comment:
Since this is now used in only one place, it could be inlined there rather than defined here, right?